test/test_runner.go

```diff
@@ -86,15 +86,23 @@ func NewLogCollectionHook() *LogCollectionHook {
 
 func (h *LogCollectionHook) Run(e *zerolog.Event, level zerolog.Level, msg string) {
 	if level != zerolog.NoLevel {
-		msg = strings.Replace(
+		logMsg := strings.Replace(
 			msg,
 			"LOG:",
 			"",
 			1,
 		)
 		re := regexp.MustCompile("\"(.*)\"")
```

Comment:
This should be moved into a global, it should only be compiled once

Reply:
Good catch :100:
Fixed in https://github.com/onflow/cadence-tools/pull/138/commits/54251a8d6c00479dc224ae27c70c68a25c085e52